.rubocop.yml

```diff
@@ -13,13 +13,25 @@ AllCops:
   DisplayStyleGuide: true
   ExtraDetails: true
 
+Layout/LineLength:
+  Max: 120
+
+Metrics/AbcSize:
+  Exclude:
+    - test/**/*.rb
+
+Metrics/ClassLength:
+  Exclude:
+    - test/**/*.rb
+
+Metrics/MethodLength:
+  Exclude:
+    - test/**/*.rb
```

Comment:
We already had many exceptions for tests in the TODO file, but IMO these shouldn't apply to tests. Feel free to push back on this and I can revert them. In many cases the tests just need to be that complex/long to exercise their subject.